### ISSUE TITLE
Support for NVLM

### ISSUE THREAD

**Issue by mitar** [ollama/ollama#7080]
NVLM is model from Nvidia: https://nvlm-project.github.io/

**Comment by TKaluza** [comment]
So it would run a A100 /H100?

**Comment by nonetrix** [comment]
Have both of you not heard of quantization or something? Also parameters are not the same as gigabytes

**Comment by nonetrix** [comment]
I'm able to run it on 4 bits with 16GBs of VRAM and 64GBs of RAM adding up to 80GBs, but I think you are getting parameters and gigabytes confused. The model is presumably trained at FP16 and the accuracy can be lowered with quantization to FP8, FP4, and so on to fit in smaller amounts of memory sacrificing accuracy

**Comment by angiopteris** [comment]
Yes, would be a great addition ! Looking forward for chatting with it

**Comment by defaultsecurity** [comment]
Looking forward to it!

**Comment by mikeazurek** [comment]
I don't think your request is even possible, because I've never seen any multi-modal AI supported by ollama, so I deduce that ollama is bound to only run purely language models, for some technical reason.

**Comment by nonetrix** [comment]
No, it does support them quite a few. This one has weird architecture I think, and Llama cpp seems quite slow to add them or more likely I think doesn't want to. I almost wish Ollama forked Llama cpp at this point
